changelog/14269.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
Should the changelog number be changed to the current PR number for the backport?

Reply:
I poked at a few other backports and they all appear to leave it the same.
E.g.: https://github.com/hashicorp/vault/pull/14829/files